astropy/units/quantity.py

```diff
@@ -1332,6 +1332,33 @@ def _decompose(self, allowscaledunits=False, bases=[]):
     # https://numpy.org/doc/stable/reference/arrays.ndarray.html#array-conversion
 
     def item(self, *args):
+        """Copy an element of an array to a scalar Quantity and return it.
+
+        Parameters
+        ----------
+        *args
+            - none: in this case, the method only works for arrays with one
+              element (``a.size == 1``), which element is copied into a
+              Quantity scalar object and returned.
+            - int_type: this argument is interpreted as a flat index into the
```

Comment:
Is `int_type` defined? Really it should be an `index_type`, I guess...  Maybe just `int`?  Also, rewrite "an index into the flattened array" (an index cannot be flat!) [I realize this comes straight from `ndarray`, but if we're adding a docstring, we might as well make it better!]